index.html

```diff
@@ -65,10 +65,30 @@ <h2>preferCurrentTab</h2>
         </dt>
         <dd>
           <p>
-            When {{MediaDevices/getDisplayMedia()}} is called, if <code>preferCurrentTab</code> is
-            <code>true</code>, the user agent SHOULD present the current tab as the most prominent
-            option. Otherwise, no such requirement is made.
+            When {{MediaDevices/getDisplayMedia()}} is called, the user agent MUST run the following
+            steps:
           </p>
+          <ol>
+            <li>
+              <p>Let <var>constraints</var> be the method's first argument.</p>
+            </li>
+            <li>
+              If <code>constraints.preferCurrentTab</code> is <code>false</code>, abort these steps
+              and continue instead with the usual set of steps associated
+              {{MediaDevices/getDisplayMedia()}}.
+            </li>
+            <li>
+              If <code>constraints.</code>{{DisplayMediaStreamOptions/selfBrowserSurface}} is
```

Comment:
Same here: `<code>|constraints|.</code>`

Reply:
That's not currently a pattern in the [parent doc](https://github.com/w3c/mediacapture-screen-share/blob/gh-pages/index.html).

Reply:
But I do see it works. Done.

Reply:
Actually, I am not sure which one is preferable. One changes the font, the other changes the font /and/ italicizes. Let's keep the pattern from getDisplayDocument's doc for now, wdys?

Reply:
Up to you. It's so short that it's okay to not have highlighting. In the parent doc though, I'd argue it has benefits.

Reply:
Done.